<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Password</class>
 <widget class="QMainWindow" name="Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Выберите уровень сложности пароля</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Простой пароль</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Надежный пароль</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сложный пароль</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Сгенерировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOpen">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="password.qrc">
          <normaloff>:/new/prefix1/open.png</normaloff>
          <disabledoff>:/new/prefix1/close.png</disabledoff>:/new/prefix1/open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="password.qrc"/>
 </resources>
 <connections/>
</ui>
